Author: Robert James Clay <jame@rocasa.us>
Description: Change configuration defaults as neccessary
Origin: vendor
Last-Update: 2016-04-11
--- a/LedgerSMB/Sysconfig.pm
+++ b/LedgerSMB/Sysconfig.pm
@@ -53,7 +53,7 @@
 our $max_post_size = 1024 * 1024;
 
 # defaults to LedgerSMB-1.3 - default spelling of cookie
-our $cookie_name = "LedgerSMB-1.3";
+our $cookie_name = "LedgerSMB-1.4";
 
 # spool directory for batch printing
 our $spool = "spool";
--- a/ledgersmb.conf.default
+++ b/ledgersmb.conf.default
@@ -35,7 +35,7 @@
 # Spelling of cookie name for this instance of LedgerSMB.
 # Used to keep multiple instances and/or different versions
 # logged in at the same time
-cookie_name = LedgerSMB-1.3
+cookie_name = LedgerSMB-1.4
 
 # This is the string we look for in the failed connection error to determine
 # if the database was not found.  For English-language locales, this can be
@@ -98,7 +98,7 @@
 # Sets database connection parameters.  Note that default_db can be left blank
 # if you want to force people to enter a company name at login.
 port = 5432
-default_db = lsmb13
+default_db = lsmb14
 host = localhost
 db_namespace = public
 # sslmode can be require, allow, prefer, or disable.  Defaults to prefer.
