Author: Robert James Clay <jame@rocasa.us>
Description: Correct path for Bash in tools scripts.
Origin: vendor
Last-Update: 2016-04-14
--- a/tools/regen_db_docs.sh
+++ b/tools/regen_db_docs.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/usr/bash
 
 #  PostgreSQL Autodoc - http://www.rbt.ca/autodoc/
 
--- a/tools/system/lsmbstart
+++ b/tools/system/lsmbstart
@@ -1,5 +1,5 @@
-#!/usr/bin/bash
+#!/usr/bash
 
-cd /usr/share/ledgersmb
+cd /usr/share/ledgersmb-1.4
 plackup -s FCGI -D --listen /var/run/httpd/fastcgi/lsmb.sock
 
--- a/tools/system/lsmbstop
+++ b/tools/system/lsmbstop
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/usr/bash
 
 killall perl-fcgi-pm
 
